brvs-rules/src/main/java/iroha/validation/rules/impl/BlockRule.java

```diff
@@ -0,0 +1,104 @@
+package iroha.validation.rules.impl;
+
+import iroha.protocol.Commands;
+import iroha.protocol.Commands.Command;
+import iroha.protocol.QryResponses.QueryResponse;
+import iroha.protocol.TransactionOuterClass.Transaction;
+import iroha.validation.rules.Rule;
+import java.io.StringReader;
+import java.security.KeyPair;
+import javax.json.Json;
+import javax.json.JsonReader;
+import jp.co.soramitsu.iroha.java.IrohaAPI;
+import jp.co.soramitsu.iroha.java.Query;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+public class BlockRule implements Rule {
+
+  private static final Logger logger = LoggerFactory.getLogger(BlockRule.class);
+
+  private IrohaAPI irohaAPI;
+  private String blockerAccount = "blocker@trading";
+  private String brvsAccountId;
+  private KeyPair keyPair;
+
+  public BlockRule(IrohaAPI irohaAPI, String brvsAccountId,
+      KeyPair keyPair) {
+    this.irohaAPI = irohaAPI;
+    this.brvsAccountId = brvsAccountId;
+    this.keyPair = keyPair;
+
+  }
+
+
+  private boolean validateTransfer(Commands.Command transfer, Transaction tx) {
+    String creator = tx.getPayload().getReducedPayload().getCreatorAccountId();
+    String from = transfer.getTransferAsset().getSrcAccountId();
+    logger.info("Creator: {}\nFrom: {}", creator, from);
+    if (!creator.equals(from)) {
+      return true;
+    }
+    String assetId = transfer.getTransferAsset().getAssetId();
+    double amount = Double.parseDouble(transfer.getTransferAsset().getAmount());
+    logger.info("Asset: {}\nAmount: {}", assetId, amount);
+
+    String key = assetId.replace("#", "___");
```

Comment:
this is not transparent as well, should be passed as an argument I think. Different projects may want to use different separators etc